tools/scripts/gen-maven-files.sh

```diff
@@ -7,7 +7,7 @@ echo "*********************************"
 
 . ./tools/scripts/variables.sh
 
-ossrhUsername=$(grep -Po '<username>\K[^\<]+' ./libs/java/dal/db/.devcontainer/maven-settings.xml)
+ossrhUsername=$(grep -Po '<username>\K[^\<]+' ./libs/java/dal/db/.devcontainer/maven-settings.xml 2>/dev/null)
```

Comment:
What does this do?

Reply:
Sends stderr (2) to /dev/null (rather than screen) so error doesn't appear on the terminal saying grep cant find the file that it's trying to read the variable from. We know the file/variable doesn't exist when it prompts for it.